pkg/stockings/stock.go

```diff
@@ -0,0 +1,159 @@
+package stockings
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+)
+
+/*
+	Could look to make this an interface as we move forward with different stock apis
+	ie make different structs for td, iex, rh apis and implement each of these methods for the
+	different apis via StockAPI interface.
+
+*/
+type Stock struct {
+	Symbol        string  `json:"symbol"`
+	Name          string  `json:"companyName"`
```

Comment:
nit: use camelcase for json keys

Reply:
dunno what u mean by this - if you mean the `json:"..."` thats the actual name of the json field iex returns. do u want me to rename Symbol to StockTicker and name to CompanyName?

Reply:
ohhh, I see what you mean - in that case could you rename the Stock struct to something with `Iex` in it? I'm planning on having a generic `Stock` struct later on, which would encompass a more general type that could be used across different brokerages and apis

Reply:
resolved